app/controllers/invitations_controller.rb

```diff
@@ -22,11 +22,10 @@ def index
   end
 
   def manage
-    if params[:status].blank? || !['unsent', 'unredeemed', 'redeemed'].include?(params[:status])
-      @invitations = @user.invitations
-    else
-      @invitations = @user.invitations.send(params[:status])
-    end
+    @invitations = @user.invitations
```

Comment:
Maybe in line with what you've done elsewhere, this might work as a case too?
```
@invitations = 
  case params[:status]
  when "unsent"
    @user.invitations.unsent
  when "unredeemed"
    @user.invitations.unredeemed
  when "redeemed"
    @user.invitations.redeemed
  else
    @user.invitations
  end
```